protocol.html

```diff
@@ -708,6 +755,8 @@ <h3 property="schema:name">Writing Resources</h3>
 
                   <p><span about="" id="server-protect-containment" rel="spec:requirement" resource="#server-protect-containment"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> NOT allow HTTP <code>POST</code>, <code>PUT</code> and <code>PATCH</code> to update a container’s containment triples; if the server receives such a request, it <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> respond with a <code>409</code> status code.</span></span> [<a href="https://github.com/solid/specification/issues/40#issuecomment-573358652" rel="cito:citesAsSourceDocument">Source</a>]</p>
 
+                  <p><span about="" id="server-protect-authoritative-resource-data" rel="spec:requirement" resource="#server-protect-authoritative-resource-data"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> NOT allow HTTP <code>POST</code>, <code>PUT</code> and <code>PATCH</code> to update the <a href="#authoritative-resource-data">authoritative resource data</a>; if the server receives such a request, it MUST respond with a <code>409</code> status code.</span></span> [<a href="https://github.com/solid/specification/issues/227#issuecomment-919312592" rel="cito:citesAsSourceDocument">Source</a>]</p>
```

Comment:
How does one distinguish between the `dcterms:modified` triples that are "authoritative metadata" and the `dcterms:modified` triples that a client supplies? Or are all attempts (malicious or benign) to use `dcterms:modified` as a predicate to be rejected by a server?

Reply:
Reject all attempts. Authoritative information about contained resources in the container description are strictly claimed and protected by the server. In the same way containment statements are protected by the server. Moreover, thus far there is no compelling case for clients to update these statements about contained resources in the container description. There are no additional restrictions on clients attempting to claim such statements anywhere else in the storage.